libraries/WiFi/examples/WiFiProv/README.md

```diff
@@ -63,24 +63,15 @@ WiFi.beginProvision(scheme prov_scheme, wifi_prov_scheme_event_handler_t scheme_
     * scheme_event_handler = WIFI_PROV_EVENT_HANDLER_NONE
     * security = WIFI_PROV_SECURITY_1
     * pop = "abcd1234"
-    * service_name = "WiFi_XXX" 
+    * service_name = "WiFi_XXX", where XXX are the last 3 bytes of MAC address of your device
```

Comment:
As you have mentioned about the MAC address above, I think it is okay to remove this description from here

Reply:
Ok